components/TheLabOptionsMobile.vue

```diff
@@ -0,0 +1,133 @@
+<script setup>
+import VFlexibleLink from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VFlexibleLink.vue'
+
+const props = defineProps({
+  index: {
+    type: Number,
+    default: 0,
+  },
+  tiers: {
+    type: Array,
+    default: null,
+  },
+  options: {
+    type: Array,
+    default: null,
+  },
+})
+
+
+const labelsToHideArray = ['Merch', 'Swag']
+/**
+ * Function to remove items in labelsToHideArray 
+ */
+const hideLabels = (label) => {
```

Comment:
It's a little confusing that the function called `hideLabels` returns `false` if the label is in the array of `labelsToHideArray`. I would recommend either changing the function name _or_ adding a `hidden: boolean` property to the `option` object to replace the `labelsToHideArray` array and `hideLabels` function.

Reply:
yeah, adding property to the option object was a WAY better way to handle this! Thanks!